certification/runtime/config_test.go

```diff
@@ -15,6 +16,10 @@ var _ = Describe("Viper to Runtime Config", func() {
 		baseViperCfg = viper.New()
 		expectedRuntimeCfg = &Config{}
 
+		if val, ok := os.LookupEnv("KUBECONFIG"); ok {
```

Comment:
NIT: good to merge without this change.
```suggestion
		if val, isSet := os.LookupEnv("KUBECONFIG"); isSet {
```

Reply:
Happens in a couple of places, but because it's a nit, I'm only mentioning it here.

Reply:
Updated.